opal/corelib/basic_object.rb

```diff
@@ -48,21 +48,21 @@ def !=(other)
 
   def instance_eval(*args, &block)
     if block.nil? && `!!Opal.compile`
-      Kernel.raise ArgumentError, "wrong number of arguments (0 for 1..3)" unless (1..3).cover? args.size
+      ::Kernel.raise ::ArgumentError, "wrong number of arguments (0 for 1..3)" unless (1..3).cover? args.size
```

Comment:
no, it's because from inside BasicObject you can't access top level constants… except by making the lookup _qualified_ with the leading double-dots and ignore `Module.nesting`.

See also the changes in `native.rb`